src/WS2812FX/WS2812FX_FastLed.h

```diff
@@ -821,6 +821,11 @@ class WS2812FX
 
   // Expose the random wheel index function for effects
   uint8_t get_random_wheel_index(uint8_t, uint8_t);
```

Comment:
We should move this functions to the EffectHelper class as they may be common for more effects though.

**Tasks:**

1. **Move the follwoing functions to EffectHelper:**

- [ ] get_random_wheel_index
- [ ] triwave16
- [ ] quadwave16
- [ ] cubicwave16
- [ ] ease16InOutQuad(uint16_t i)
- [ ] ease16OutQuad(uint16_t i)
- [ ] ease16InQuad(uint16_t i)

2. **Check all Files in the "/src/WS2812FX/effects** Folder if they use (or should use) this function instead of an own implementation.
3. Correct these files / classes to make use of these function.